Dockerfile

```diff
@@ -1,3 +1,10 @@
+FROM alpine:latest as alpine
+RUN apk --update add unzip && rm -rf /var/cache/apk/*
+WORKDIR /tmp
+COPY docker/MapStore-*.war mapstore.war
```

Comment:
Note to the reviewer: the war file is provided by https://github.com/georchestra/mapstore2-georchestra/blob/59e7d1d209089aebc3af00fdc3c89d063c713c42/.github/workflows/mapstore.yml#L96